docs/topics/signalen-and-standard-for-public-code.md

```diff
@@ -1,196 +1,222 @@
+Any non-open standards used MUST be recorded clearly as such in the documentation. | N/A |
+Any standard chosen for use within the codebase MUST be listed in the documentation with a link to where it is available. | yes | [standards](https://github.com/Amsterdam/signals/blob/master/docs/CONTRIBUTING.md#standards)
+Any non-open standards chosen for use within the codebase MUST NOT hinder collaboration and reuse. | N/A |
+If no existing open standard is available, effort SHOULD be put into developing one. | N/A |
 Standards that are machine testable SHOULD be preferred over those that are not. | N/A |
-Functionality using features from a non-open standard (one that doesn’t meet the Open Source  Initiative Open Standard Requirements) MAY be provided if necessary, but only in addition to compliant features. |  |
-All non-compliant standards used MUST be recorded clearly in the documentation. | N/A |
-The codebase SHOULD contain a list of all the standards used with links to where they are available. | no | Created [backend#65](https://github.com/Signalen/backend/issues/65).
 
 ## [Use continuous integration](https://standard.publiccode.net/criteria/continuous-integration.html)
 
-- [ ] compliant with this criterion.
+- [ ] criterion met.
 
 Requirement | meets | links and notes
 -----|-----|-----
-All functionality in the source code MUST have automated tests. | mostly | In addition to unit testing, there are end-to-end tests which run on master and release branches (frontend); there are coverage reports, results require manual download.
+All functionality in the source code MUST have automated tests. |  | Would like to see code coverage published
 Contributions MUST pass all automated tests before they are admitted into the codebase. | yes |
-Contributions MUST be small. | yes | Each pull request contains a single feature or fix, but this policy is not explicit in CONTRIBUTING.md nor explictily checked by reviewer.
+The codebase MUST have guidelines explaining how to structure contributions. | yes | [CONTRIBUTING](https://github.com/Amsterdam/signals/blob/master/docs/CONTRIBUTING.md)
 The codebase MUST have active contributors. | yes | Amsterdam & VNG, [pulse](https://github.com/Amsterdam/signals/pulse/monthly)
-Source code test and documentation coverage SHOULD be monitored. | no | Coverage statics are not monitored but tests will fail when coverage is below a threshold.
-Policy and documentation MAY have testing for consistency with the source and vice versa. |  |
-Policy and documentation MAY have testing for style and broken links. |  |
+The codebase guidelines SHOULD state that each contribution should focus on a single issue. | yes | [CONTRIBUTING](https://github.com/Amsterdam/signals/blob/master/docs/CONTRIBUTING.md) (could be more explicit)
+Source code test and documentation coverage SHOULD be monitored. |  | Coverage statics are not monitored but tests will fail when coverage is below a threshold.
```

Comment:
I think this is a a matter of definitions, we do actively make sure we stay above the minimum threshold. There are no graphs over time if that is meant, but those would not be interesting (perpetually hovering just above the threshold we set).

Reply:
The pull request template has the individual check that code coverage is at least 85%; I don't see an alert or similar automated check if the code coverage has dropped below the threshold ... am I simply over-looking it?

Reply:
https://github.com/Amsterdam/signals/blob/main/api/app/tox.ini#L64 our current threshold below which the CI fails.

Reply:
Thanks, indeed I was missing it.